libs/go/zmscli/role.go

```diff
@@ -231,7 +246,13 @@ func (cli Zms) AddMembers(dn string, rn string, members []string) (*string, erro
 	} else {
 		s = "[Added to " + rn + ": " + strings.Join(ms, ",") + "]"
 	}
-	return cli.switchOverFormats(s)
+
+	message := SuccessMessage{
+		Status:  200,
+		Message: s,
+	}
+
```

Comment:
For "old Yaml" we just printed out the error message without format (unlike in ShowRole function).
So instead of wrapping this logic as the "manual yaml converter", I passed the regular cli.buildYAMLOutput so it will appear in Yaml format.